{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "40789113",
  "multipleFaultNumbers": null,
  "date": "2020-12-04T14:06:04",
  "region": "Bolton",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-12-04T14:06:04",
  "information": "",
  "actualTimeOfRestoration": "2020-12-04T14:06:04",
  "outageCentrePoint": {
    "lat": 53.549611,
    "lng": -2.45118
  },
  "outageLocations": null,
  "AffectedPostcodes": " BL4 0LN, BL4 0LP, BL4 0LR, BL4 0LS, BL4 0LW, BL4 0NN, BL5 1AL, BL5 1AN",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}